### DISCUSSION TITLE
How to make selfbot play my account even though I didn't turn my pc on

### DISCUSSION THREAD

**Discussion by SyHaqi** [rigwild/discord-self-bot-console#17] [Q&A] [answered]
I am new, very new in this coding world to be honest. I don't really understand the guide and the discussion. What I want is just to make selfbot play discord bot game on my stead without me opening my pc. It will be for every 5 minutes the selfbot will send a message and keep going until I turn of the loop. Is it possible? and how to do it?

**Comment by rigwild** [comment, marked as answer]
Idk what your game is? If it's send a message, you can do it like this:

```js
{
  id()
  let channelId = cid
  let message = 'Hello'

  var loop = true
  let count = 0
  while (loop) {
    const sentMessage = await api.sendMessage(channelId, message)
    console.log(`Sent ${++count} messages`)
    await delay(5 * 60 * 1000) // 5 minutes
  }
}
```

To stop it:

```js
loop = false
```

If you want to run it without your pc, you need a server. I can not assist with that.
You can have a free one on Google Cloud Compute Engine https://cloud.google.com/free
See also https://github.com/rigwild/discord-self-bot-console/discussions/4#discussioncomment-1438231

**Reply by SyHaqi** [reply]
Thanks, btw the bot game that i am playing is Epic RPG
